File: app/src/main/java/br/com/onibusapp/onibusapp/ui/favoritos/FavoritosContract.java
package br.com.onibusapp.onibusapp.ui.favoritos;


import java.util.List;

import br.com.onibusapp.onibusapp.data.dominio.Favorito;

/**
 * Created by diego on 16/07/2018.
 */

public interface FavoritosContract {

    interface View {

        void carregar(List<Favorito> favoritos);

        void atualizar(Integer posicao);
    }

    interface Presenter {

        void apagar(Integer posicao);

        void carregarMapa(Integer posicao);
    }
}

File: app/src/main/java/br/com/onibusapp/onibusapp/ui/favoritos/FavoritosContract.kt
package br.com.onibusapp.onibusapp.ui.favoritos


import br.com.onibusapp.onibusapp.data.dominio.Favorito

/**
 * Created by diego on 16/07/2018.
 */

interface FavoritosContract {

    interface View {

        fun carregar(favoritos: List<Favorito>)

        fun atualizar(posicao: Int?)
    }

    interface Presenter {

        fun apagar(posicao: Int?)

        fun carregarMapa(posicao: Int?)
    }
}
